phpcs.xml

```diff
@@ -0,0 +1,267 @@
+    <!-- This is impractical when commenting code out. -->
+    <exclude name="Drupal.Commenting.InlineComment.InvalidEndChar" />
+    <!-- We (rarely) use comments as "headings" for multiple functions. -->
+    <exclude name="Drupal.Commenting.InlineComment.SpacingAfter" />
+    <!--
+    This disallows indentation in comments, even though it can sometimes be
+    helpful for structured explanations.
+
+    @see \Drupal\search_api\Plugin\search_api\processor\ContentAccess::addNodeAccess()
+    -->
+    <exclude name="Drupal.Commenting.InlineComment.SpacingBefore" />
+  </rule>
+  <rule ref="Drupal.Commenting.VariableComment">
+    <!-- This finds false positives when @code is used. -->
+    <exclude name="Drupal.Commenting.VariableComment.VarOrder"/>
+  </rule>
+  <rule ref="Drupal.Commenting.PostStatementComment"/>
+  <rule ref="Drupal.ControlStructures.ElseIf"/>
+  <rule ref="Drupal.ControlStructures.ControlSignature"/>
+  <rule ref="Drupal.ControlStructures.InlineControlStructure"/>
+  <rule ref="Drupal.Files.EndFileNewline"/>
+  <rule ref="Drupal.Files.FileEncoding"/>
+  <rule ref="Drupal.Files.TxtFileLineLength"/>
+  <rule ref="Drupal.Formatting.MultiLineAssignment"/>
+  <rule ref="Drupal.Formatting.SpaceInlineIf"/>
+  <rule ref="Drupal.Formatting.SpaceUnaryOperator"/>
+  <rule ref="Drupal.Functions.DiscouragedFunctions"/>
+  <rule ref="Drupal.Functions.FunctionDeclaration"/>
+  <rule ref="Drupal.InfoFiles.AutoAddedKeys"/>
+  <rule ref="Drupal.InfoFiles.ClassFiles"/>
+  <rule ref="Drupal.InfoFiles.DuplicateEntry"/>
+  <rule ref="Drupal.InfoFiles.Required"/>
+  <rule ref="Drupal.Methods.MethodDeclaration"/>
+  <rule ref="Drupal.NamingConventions.ValidVariableName">
+    <!-- This interferes with the stored entity properties. -->
+    <exclude name="Drupal.NamingConventions.ValidVariableName.LowerCamelName"/>
+  </rule>
+  <rule ref="Drupal.Scope.MethodScope"/>
+  <rule ref="Drupal.Semantics.EmptyInstall"/>
+  <rule ref="Drupal.Semantics.FunctionAlias"/>
+  <rule ref="Drupal.Semantics.FunctionT"/>
+  <rule ref="Drupal.Semantics.FunctionWatchdog"/>
+  <rule ref="Drupal.Semantics.InstallHooks"/>
+  <rule ref="Drupal.Semantics.LStringTranslatable"/>
+  <rule ref="Drupal.Semantics.PregSecurity"/>
+  <rule ref="Drupal.Semantics.TInHookMenu"/>
+  <rule ref="Drupal.Semantics.TInHookSchema"/>
+  <rule ref="Drupal.WhiteSpace.CloseBracketSpacing"/>
+  <rule ref="Drupal.WhiteSpace.Comma"/>
+  <rule ref="Drupal.WhiteSpace.EmptyLines"/>
+  <rule ref="Drupal.WhiteSpace.Namespace"/>
+  <rule ref="Drupal.WhiteSpace.ObjectOperatorIndent"/>
+  <rule ref="Drupal.WhiteSpace.ObjectOperatorSpacing"/>
+  <rule ref="Drupal.WhiteSpace.OpenBracketSpacing"/>
+  <rule ref="Drupal.WhiteSpace.OpenTagNewline"/>
+  <rule ref="Drupal.WhiteSpace.OperatorSpacing"/>
+  <rule ref="Drupal.WhiteSpace.ScopeClosingBrace"/>
+  <rule ref="Drupal.WhiteSpace.ScopeIndent"/>
+
+  <!-- Drupal Practice sniffs -->
+  <rule ref="DrupalPractice.Commenting.ExpectedException"/>
```

Comment:
With this phpcs ruleset which is much more relaxed (only takes one sniff from the DrupalPractice ruleset), most of the fixes I did in commit a072c3ab72d495ac628023ea3edabab632409bf3 are not really required.
Let me know if you want me to revert those changes.